### ISSUE TITLE
Remove Welcome Screen for updates

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#586]
We need to remove the Welcome Screen for all updates.  We will add it back for major updates when it provides value such as directing a user to a new feature they should configure.  

See also issue #484.

**Comment by michaeltorbert** [comment]
For the time being we'll simply remove it completely for updates unless they're updating from pre-2.3.9

**Comment by michaeltorbert** [comment]
Ok this should only happen on new installs... We can always manually call it as needed for large major updates in the future.

**Comment by wpsmort** [comment]
@michaeltorbert I get the Welcome Screen when updating from 2.3.9.2 to 2.3.10.2.  

To replicate this I installed 2.3.9.2 and made sure the last active version in the database said 2.3.9.2.  I then deactivated AIOSEOP, deleted the directory, uploaded the 2.3.10.2 directory to the plugins directory and went to my site and activated AIOSEOP.  The Welcome Screen appeared upon activation.  I was also able to replicate this going from 2.3.10.1 too.
